radis/misc/utils.py

```diff
@@ -96,6 +96,15 @@ def __del__(self):
 # ==============================================================================
 
 
+class Default:
+    """ Returns argument `unit` with an appended `_by_default` string
+    """
+
+    def __new__(cls, unit):
+        cls.unit = unit + "_by_default"
+        return cls.unit
+
+
```

Comment:
In this version you'd still keep the "_by_default" string, which I find a bit verbosy.

What do you think of this:

```suggestion
class Default:
    """ Contains a value
    
    Examples
    --------
    
    ::
    
        a = Default("42")
        isinstance(a, Default)
        >>> True
        
        a.value
        >>> 42
    """

    def __init__(self, value):
        self.value= value

```

Which also has the advantage to work for anything (i.e. : `Default(1.mbar)` if we ever need it ) 

Could event implement the "__eq__" method so you can get "a == 42 >>> True" 

And then, when entering the function we can evaluate all the Default arguments as `a = a.value`

Reply:
Ah I see. The main reason why I implemented it with the `__new__` magic method was to avoid 
> when entering the function we can evaluate all the Default arguments as a = a.value

But I think you're right about the multi-functionality that your implementation has to offer! I'll change it right away.